yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -318,36 +319,35 @@ Internet-Draft    Updated YANG Module Revision Handling    February 2024
    refine the rules for permissible changes when a new YANG module
    revision is created.
 
-   While a new module revision SHOULD NOT contain NBC changes, they are
-   allowed.  For example:
+   New module revision SHOULD NOT contain NBC changes because they often
```

Comment:
New module revisions? @rgwilton

Reply:
Yes, please can you include with your updates?

Reply:
Will do